platform_plugin_teams/apps.py

```diff
@@ -11,3 +11,26 @@ class PlatformPluginTeamsConfig(AppConfig):
     """
 
     name = "platform_plugin_teams"
+    verbose_name = "Custom Teams API Plugin"
+
+    plugin_app = {
+        "url_config": {
+            "lms.djangoapp": {
+                "namespace": "platform-plugin-teams",
+                "regex": r"^platform-plugin-teams/",
```

Comment:
Currently, the `course_id` is being passed as a query parameter or a body parameter, but we could include it as a path parameter.

Reply:
I mentioned the course authoring MFE as a reference since topics are created from there. Can we check how it's done there? So we follow the same conventions

Reply:
This is the endpoint used for creating and deleting a topic:

```bash
Request URL: <cms_host>/api/contentstore/v0/advanced_settings/course-v1:eduNEXT+Teams+Demo
Request Method: PATCH
```

They receive the `course_id` from the route, we could do something similar, perhaps:

```bash
POST <cms_host>/platform-plugin-teams/<course_id>/api/topics/
DELETE <cms_host>/platform-plugin-teams/<course_id>/api/topics/<topic_id>/
```
What do you think? @mariajgrimaldi